python/hopsworks/flink_cluster.py

```diff
@@ -0,0 +1,45 @@
+#
+#   Copyright 2023 Hopsworks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import time
+from hopsworks import job
+
+
+class FlinkCluster(job.Job):
+    pass
+
+    def start(self, start_time_out=600):
+        execution = super().run()
+
+        polling_time = 0
+        while polling_time < start_time_out:
+            execution = self._execution_api._get(self, execution.id)
+            if execution.state == "RUNNING":
```

Comment:
We should have an else statement here how handle if the job failed while starting up. Otherwise the user will wait 10 minutes and then get the exception thrown on line 43.